.rubocop.yml

```diff
@@ -22,3 +22,6 @@ Naming/AccessorMethodName:
 
 Metrics/AbcSize:
   Enabled: false
+
+Style/EachWithObject:
+  Enabled: false
```

Comment:
Is `each_with_object` used?

Reply:
Rubocop would yell at me when I used inject instead of each_with_object